book.md

```diff
@@ -177,7 +177,146 @@ vulnerabilities, along with their mitigations, presented in a rough
 chronological order of their appearance, and, therefore, complexity.
 
 ## Exploitation primitives
-\missingcontent{Discuss exploitation primitives}
+
+Newcomers to the area of software security may find themselves lost in a large
+number of blog posts and other publications describing specific memory
```

Comment:
Possible simplification of "a large number of" to "many".

Reply:
Good point! I'll make the change.